scripts/asm-installer/cloudbuild-master.yaml

```diff
@@ -22,6 +22,18 @@ steps:
       - '${_SCRIPT_NAME}'
       - '${_ASM_VM_SCRIPT_NAME}'
 
+  - name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
+    dir: 'scripts/asm-installer'
+    id: 'script-version-check'
+    entrypoint: '/bin/bash'
+    args:
+      - '-c'
+      - >
+        ./tests/script_version_check.sh
+    env:
+      - 'INSTALL_ASM_SCRIPT=${_SCRIPT_NAME}'
+      - 'ASM_VM_SCRIPT=${_ASM_VM_SCRIPT_NAME}'
+
```

Comment:
Don't parameterize these, just hardcode them in the script.

Reply:
Done.